EXT_LaserEnvelope.md

```diff
@@ -0,0 +1,68 @@
+Domain-Specific Naming Conventions for a laser envelope profile
+===============================================================
+
+openPMD extension name: `LaserEnvelope`
+
+
+Introduction
+------------
+
+This extension is specifically designed for the modeling laser beam propagation.
+
+Mesh Based Records (Fields)
+---------------------------
+
+### Naming Conventions for `mesh record`s (field records)
+
+The laser pulse is represented as a single complex field $\mathcal{E}$, describing the envelope of the laser electric field in the paraxial approximation:
+
+```math
+   \begin{aligned}
+   E_x &= \operatorname{Re}\left( \mathcal{E} e^{-i\omega_0t}p_x\right)\\
+   E_y &= \operatorname{Re}\left( \mathcal{E} e^{-i\omega_0t}p_y\right)\\
+   E_z &= 0\\
+   B_x &= -c E_y\\
+   B_y &= c E_x\\
+   B_z &= 0\end{aligned}
```

Comment:
```suggestion
\end{aligned}
```
Would it be okay to only specify `E_x` and `E_y` here? In principle, `E_z` could be non-zero (even in the paraxial approximation, I think).

Reply:
I think non-zero Ez would be a correction to the paraxial approximation, e.g. in https://journals.aps.org/pre/abstract/10.1103/PhysRevE.58.3719. In any case, I agree it is better not to specify that these are 0 because they do not have to be.